logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -72,7 +72,7 @@ public void blockCreationFailure(final Exception cause) {
     log.error("Error during block creation", cause);
   }
 
-  public void attestationFailure(final IllegalArgumentException cause) {
+  public void attestationFailure(final Throwable cause) {
```

Comment:
Might be a good idea to name this as dutyFailure or something.

Reply:
That message is specifically for failing to create an attestation - there's a separate status event for failing to create a block.

Reply:
Then we should probably change the log.warn() string below at some point.